examples/autoscaler-webhook/dockerfile

```diff
@@ -0,0 +1,24 @@
+# Gather dependencies and build the executable
```

Comment:
Minor nits - should be `Dockerfile`.

Also need the Apache licence at the top.